packages/sanity/src/_internal/cli/actions/schema/extractAction.ts

```diff
@@ -0,0 +1,70 @@
+import {type CliCommandArguments, type CliCommandContext} from '@sanity/cli'
+import {writeFile} from 'fs/promises'
+import {dirname, join} from 'path'
+import readPkgUp from 'read-pkg-up'
+import {Worker} from 'worker_threads'
```

Comment:
Prefer `node:` prefixed imports for new code

```suggestion
import {writeFile} from 'node:fs/promises'
import {dirname, join} from 'node:path'
import readPkgUp from 'read-pkg-up'
import {Worker} from 'node:worker_threads'
```